spec/services/accessories_child_updater_spec.rb

```diff
@@ -63,7 +63,7 @@
       end
 
       it "moves the child to canceled" do
-        expect(child_order_detail).to be_canceled
+        expect(child_order_detail.state).to eq "canceled"
```

Comment:
Oops, yeah, that was to avoid a name collision before I renamed to `is_canceled?`, forgot to revert that one.